### ISSUE TITLE
If no dpad allow the option to have external DPAD

### ISSUE THREAD

**Issue by Invictaz** [sonik-br/xid_wheel_adapter#2]
I'm still working on my Formula GP and it's adapter. Soldering the adapter is a bit postponed but it's coming. However it does not have a dpad.

So if I want to have dpad on the Xbox I might want to use a controller in another controller port for that. 

Is that an option to have some buttons on a controller and the rest on the wheel?

**Comment by sonik-br** [comment]
Unfortunately it's not possible.
You would need to add and code support for it.

One possibility would be to map one of the wheel's buttons to act like a shift function. When it's pressed, the other buttons would act as the dpad.

The lib support for usb hub is hit and miss. You would need to try. And the current code allows just a single controller connected, would also require some code adjustments.

You can also add external analog stick or push buttons and code it to act as a dpad.

**Comment by Invictaz** [comment]
In fact it is possible because in GIMX (I use version 8) you simply can map gas/throttle to the real pedals and the menu buttons to the attached controller. GIMX uses the USB output to the console (via serial adapter) and an extra controller connected to it's USB port.

Also in CronusMax with X-AIM you can do this. It uses the authentication controller to the CronusMax for secondary input.

My wheel does not have any ports available. The only one is the RJ-11 style port for the pedals..

**Comment by sonik-br** [comment]
Yes it's possible. I've even suggested some ways to implement it.
